Woodoku/src/game.py

```diff
@@ -1,17 +1,48 @@
 import random
 from typing import List
 
+import yaml
+
 from Entity.WoodokuBoard import WoodokuBoard
 from Entity.WoodokuShape import WoodokuShape
 from UI.CommandLineUI import CommandLineUI
 from UI.UIInterface import UIInterface
 
-CONFIG_FILE = ''
+CONFIG_FILE = './config.yaml'
 NUM_SHAPES = 3
 
 
 def read_shapes_from_file(filepath: str) -> List[WoodokuShape]:
-    pass
+    raw_shapes = []
+    with open(filepath) as config:
+        raw_shapes_list = yaml.safe_load(config)['raw_shapes']
+        for row in raw_shapes_list:
+            raw_shapes.append(WoodokuShape(row))
+
```

Comment:
Since list is not hashable, the error occurs in WoodokuShape.__init__
<img width="957" alt="Screen Shot 2022-07-22 at 11 19 01 PM" src="https://user-images.githubusercontent.com/67870437/180588657-74ba56b3-4599-4465-a0d6-740db750524b.png">

Reply:
Nice catch. Just updated.